examples/plot_sleep_staging.py

```diff
@@ -96,22 +96,27 @@
 
 
 mapping = {  # We merge stages 3 and 4 following AASM standards.
-    'Sleep stage W': 0,
-    'Sleep stage 1': 1,
-    'Sleep stage 2': 2,
-    'Sleep stage 3': 3,
-    'Sleep stage 4': 3,
-    'Sleep stage R': 4
+    "Sleep stage W": 0,
+    "Sleep stage 1": 1,
+    "Sleep stage 2": 2,
+    "Sleep stage 3": 3,
+    "Sleep stage 4": 3,
+    "Sleep stage R": 4,
```

Comment:
you should revert changes in this file.

Reply:
I'm not sure I understand what this means

Reply:
you should not have touched this file in this PR.

Reply:
absolutely - I just made sure this script is up-to-date with braindecode master so we don't inadvertently change it.